examples/backends/sglang/launch/disagg_dp_attn.sh

```
@@ -0,0 +1,113 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Disaggregated DP-attention serving on two shared GPUs. Each prefill/decode
+# worker owns two DP ranks over the same GPU pair, which keeps this smoke test
+# representative without requiring a separate two-GPU pair for each role.
+# GPUs: 2
+
+set -e
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../common/gpu_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/gpu_utils.sh was not specified as input (see shellcheck -x).

(SC1091)